### PR TITLE
a playbook to do ml2 migration cleanup

### PR DESCRIPTION
**Pull request by sandfish8**
This can stay in the 1.0.x branch, it should not go to master

### CODE REVIEW

File: playbooks/ovs_2_ml2_cleanup_remnants.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+
+# a cleanup script to run after a host has been migrated from OVS to ml2 / linux bridge
+
+- hosts: all
+  serial: 10
```
Comment (j2sol) [inline playbooks/ovs_2_ml2_cleanup_remnants.yml]: Is there a reason to do this serially?

**Comment by j2sol** [comment]
Just one question. Tests don't matter here because they won't run this playbook.

File: playbooks/ovs_2_ml2_cleanup_remnants.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - name: remove openvswitch kernel module
+      command: rmmod openvswitch
+      failed_when: False
+
+    - name: remove the phy-br-eth0 interface
+      command: ip link delete phy-br-eth0 type veth
+      when: ansible_phy_br_eth0 is defined
+
+    - name: remove OVS neutron agents
+      shell: neutron agent-list | awk '/Open vSwitch agent/ { print $2 }' | xargs neutron agent-delete
```
Comment (dlundquist) [inline playbooks/ovs_2_ml2_cleanup_remnants.yml]: you need to store stackrc here

File: playbooks/ovs_2_ml2_cleanup_remnants.yml (hunk 1 above)
Comment (sandfish8) [inline playbooks/ovs_2_ml2_cleanup_remnants.yml]: doesn't serial mean "do everything in parallel (per default behavior) but limit the batch size to 10?

Comment (j2sol) [inline playbooks/ovs_2_ml2_cleanup_remnants.yml]: Kinda yeah. Serial says run 20 hosts through the first task, then through the second task and then through the third task, etc... until no more tasks. Then run the next 20 through each task. The amount of parallelism is governed by the forks option. If serial is higher than forks, you get X forks running in parallel. If serial is lower than forks, then you get limited to the serial amount.

 I'm just curious why there is a need to make any distinction here in this play.

Comment (dlundquist) [inline playbooks/ovs_2_ml2_cleanup_remnants.yml]: Somewhere we had serial=1 defined as the default, so this was to override that. Probably not needed any more.

**Pull request merged**